index.html

```diff
@@ -2225,6 +2225,28 @@ <h2>The <dfn>record type</dfn> string</h2>
   </div>
   </section>
 
+  <section><h3>Aborting pending scan operation</h3>
+  <div>
+    To attempt to <dfn>abort a pending scan operation</dfn> on an
+    <a>environment settings object</a>, perform the following steps:
```

Comment:
Why using "environment settings object" ?

Reply:
We do the same for writing. What is the current terminology? We attached the slots to that object.

Reply:
Removed them both. The algorithm should be clear on its own.